instructure/canvas-ios
# Course Smart Search "How it works" screen

refs: [MBL-18010](https://instructure.atlassian.net/browse/MBL-18010)
affects: Student
release note: Course Smart Search "How it works" screen

Related PR(s):
https://github.com/instructure/canvas-ios/pull/2932

## Test Plan
* Make sure to activate "Smart Search" on account & course level. [See here.](https://community.canvaslms.com/t5/Smart-Search-Feature-Preview/How-to-Enable-Smart-Search/ta-p/604577)
* Login to Student app.
* Go to courses, open one of the courses. 
* You should expect a search icon shown on the top-right button. Follow on the experience.
   - Tap on question-mark icon button
   - A sheet is shown with help content about smart search
   
## Screenshot

![Help Screen](https://github.com/user-attachments/assets/ed1299bc-b3c8-4ff0-ac65-3da4033688f7)

## Video Record

https://github.com/user-attachments/assets/1e0d66f1-ffff-4043-a2aa-f96cf60464f4

## Checklist



[MBL-18010]: https://instructure.atlassian.net/browse/MBL-18010?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
